.github/workflows/tests.yml

```diff
@@ -42,3 +45,24 @@ jobs:
         shell: bash -l {0}
         run: |
           docker build --no-cache -t test .
+
+  docker-push:
+    name: docker-push
+    runs-on: "ubuntu-latest"
+    needs: tests
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: condaforgebot
+          password: ${{ secrets.DH_PASSWORD }}
```

Comment:
Does the password need to be set on this repo? Also is this the right bot to use?